docs/xks/developer-guide/networking.md

```diff
@@ -251,7 +251,9 @@ spec:
 
 ### Debugging
 
-TBD
+Common networking problems include forgetting to set up egress or ingress rules that apply for your pods - or setting them up and then having the requirements change, which then causes connection errors.
+
+Remember that you can inspect your network policies with `kubectl get networkpolicies`. If you cannot see your policy there, verify if it is actually present in your `kustomization.yaml` file.
```

Comment:
Here is a more interesting change than the language changes.

Reply:
Better than nothing but we should definitely increase this section in the future.